src/wizard/internal/analytics.tsx

```diff
@@ -36,10 +39,29 @@ const timeEnd = (key = 'current', clear = false) => {
 };
 
 export const trackStartWizard = () => {
+  timeCache = {};
+  currentStepIndex = 0;
+  success = false;
+
   timeStart(prefix);
 };
 
-export const trackStartStep = (stepIndex?: number, funnelId?: string) => {
+export const trackEndWizard = (funnelId?: string) => {
```

Comment:
```suggestion
export const trackEndWizard = (funnelId: string | undefined) => {
```

This way in stays optional in public API, but typescript will warn you if you forgot to pass it down from one internal function to another

Reply:
Updated 👍🏻